app/views/posts/show.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class='container'>
+    <h1>Post Details</h1>
+    <div class='row card mt-2 mb-2 m-0 pt-2'>
+        <h5 class='col-md-12'>
+            <%= link_to @post.title, user_post_path(@user, @post) %>
+        </h5>
```

Comment:
- [ ] Please make sure to also display the author's username along with the post title (post 1 by username) to follow the wireframe design.

![image](https://github.com/chudisoft/RubyBlog/assets/83803414/3105160e-ee40-4c4b-a498-f4814346c7db)